packages/react/src/components/IconButton/index.tsx

```diff
@@ -71,6 +77,9 @@ IconButton.propTypes = {
   // @ts-ignore
   tooltipPlacement: PropTypes.string,
   // @ts-ignore
+  tooltipVariant: PropTypes.string,
```

Comment:
Just curious - why do we have to `ts-ignore` this (but only sometimes)?

Reply:
At one point, typescript did not validate `prop-types` but we upgraded and it validated the typescript interface _and_ the prop-type values so some of that came during the change where we are trying not to break any of the existing `prop-type` values.

IMO, I don't think we need to have both `prop-types` and typescript.

Reply:
> IMO, I don't think we need to have both prop-types and typescript.

It'd be a breaking change to remove them, and would make using the components extremely difficult for non-TS users.

This said, we could just add https://github.com/milesj/babel-plugin-typescript-to-proptypes and forget about them entirely.